docs/source/requirements.rst

```diff
@@ -58,6 +58,10 @@ Kokkos 4.x
       * 22.3
       * 22.9
 
+    * * NVC++ (OpenACC) (experimental) 
+      * 23.7
+      * 24.5
```

Comment:
I don't think that is accurate.  This column says "primary tested" and we are currently testing with 23.7 in the CI
https://github.com/kokkos/kokkos/blob/cfb48fb7bf1a870a6243125d9cd66cfb0459722c/.jenkins#L185-L196

Are you running nightly builds or such on 24.5?

Reply:
I locally use 24.5 to test the OpenACC backend using Kokkos unit tests and other Kokkos benchmarks that I have.
As you said, CI uses 23.7, which is the minimum version  that works with the OpenACC backend.
Maybe, I misunderstood what "primary tested" means; I will change the primary tested version to 23.7.